pallets/manta-pay/src/test/payment.rs

```diff
@@ -160,9 +160,8 @@ where
         Some(id) => id,
         None => rng.gen(),
     };
-    // FIXME: get rid of the division after parity fixes the pallet-asset bug
-    let double_balance: u128 = rng.gen();
-    let total_free_balance = AssetValue(double_balance / 2);
+    let balance: u128 = rng.gen();
```

Comment:
Can both line of code be a function? I mean it's being used several places.
Not sure whether it's possible.

Reply:
Just made it a 1 liner.